scripts/rollup/build.js

```diff
@@ -557,14 +562,15 @@ async function createBundle(bundle, bundleType) {
   }
 
   const importSideEffects = Modules.getImportSideEffects();
-  const pureExternalModules = Object.keys(importSideEffects).filter(
+  const moduleSideEffects = Object.keys(importSideEffects).filter(
     module => !importSideEffects[module]
   );
 
   const rollupConfig = {
     input: resolvedEntry,
     treeshake: {
-      pureExternalModules,
+      propertyReadSideEffects: false,
```

Comment:
I found some code added to the production bundles that reads variables but doesn't use them, so I set propertyReadSideEffects to false to eliminate them